test/benchmark.sh

```diff
@@ -57,7 +57,7 @@ git clone https://github.com/hexojs/hexo-theme-landscape --depth=1 --quiet theme
 echo "- npm install"
 npm install --silent
 
-echo "- Import 300 posts"
+echo "- Import 500 posts"
```

Comment:
I have updated the [hexo-many-posts](https://github.com/SukkaLab/hexo-many-posts), now every post will have more swig tag & less code block.

Each post will have:

- All common markdown elements
- 5 backtick code blocks
- 5 links
- 5 images
- About 10 nunjucks tags plugins